scalableminds/webknossos
# Allow adding STL files to 3D scene

This PR adds basic mesh rendering support for STL files. You can upload an STL which will be rendered within WK. The file won't be stored within the tracing nor are interactions with the mesh supported. We should discuss potential next steps.

### URL of deployed dev instance (used for testing):
- https://stlmesh.webknossos.xyz

### Steps to test:
- use the "import stl file" menu item and upload an STL file
- it should appear at position 0, 0, 0

### Issues:
- fixes #3235 

------
